packages/itinerary-body/src/defaults/route-long-name.tsx

```diff
@@ -40,7 +45,7 @@ export default function RouteLongName({
           }}
         />
       ) : (
-        routeLongName
+        headsign || routeLongName
```

Comment:
There is a minor flaw here: if a headsign is shown but not a long name, this is rendering "67 Downtown" whereas previously it would read like "67 [...] *to* Downtown" ("to" is removed). Is this intended?

Reply:
I think "to" should still be shown in such cases.

Reply:
This was intended, but it is open to debate. I chose to hide the "to" so that the rider sees exactly what will be shown on the vehicle headsign. (assuming the GTFS is accurate) I will add this to the tech call, it's an easy change.